pipelines/common/utils/extractors/api.py

```
@@ -87,6 +87,72 @@ def get_raw_api(  # noqa: PLR0913
     return [filepath]
 
 
+def get_raw_api_paginated(  # noqa: PLR0913
+    url: str,
+    raw_filepath: str,
+    page_param_name: str,
+    page_size_param_name: str,
+    page_size: int,
+    params: dict,
+    headers: Union[None, dict] = None,
```

Ruff (0.16.1)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)